index.html

```diff
@@ -950,6 +950,9 @@ <h3>Video format support (last update July 2020)</h3>
                 stopEvent(e);
                 self.copyImage();
                 break;
+	      case 'c':
```

Comment:
Can you be sure this doesn't trigger on Ctrl + c? Otherwise this will interfere with normal copy/paste. Something like this will work:
```suggestion
	      case 'c':
	        if (e.ctrlKey || e.altKey || e.metaKey)
	          return;
	        stopEvent(e);
```